.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Docs Request
+about: File a request for incorrect or suggested docs.
+title: ''
+labels: kind/docs, triage/needs-triage
+assignees: ''
+
+---
+
+## Description
+
+<!-- Please add a detailed description of the issue or suggestion -->
+
+## Location
```

Comment:
I'm not sure we need a "docs" template for the homebrew tap since it's single purpose is to provide a formula to install `tanzu` and it's plugins. We shouldn't split documentation requests between this and the core `community-edition` repo